### ISSUE TITLE
Last character before source map line can get erased when constructing Event

### ISSUE THREAD

**Issue by PatrikLythell** [sighjs/sigh#3]
Been trying to write a postss plugin for sigh. But this function strips off the trailing } off the css files. If the lastLineIdx -1 is changed to just lastLineIdx then it works as expected. What is the reason for the -1?

Wanted to ask you before doing a pull request as this might fix something but break something else. (Will take care of (lastLine !== -1) which should be (lastLineIdx !== -1) I'm guessing in PR too.

[Event.js](https://github.com/sighjs/sigh/blob/master/src/Event.js)

```javascript
  // Set data stripping off source map comment if it exists.
  set data(value) {
    // TODO: consider applying source map (parsed via URI/file load)?
    // TODO: skip back another if empty line?
    var lastLineIdx = value.lastIndexOf('\n')
    if (lastLine !== -1) {
      var lastLine = value.substr(lastLineIdx + 1)
      if (sourceMapRegex.test(lastLine))
        value = value.substr(0, lastLineIdx - 1)
    }

    this._data = value
  }
```

**Comment by ohjames** [comment]
Nice catch. If you open a PR I'll release a new version with the fix right away. Thanks for choosing sigh, the best js build system.

**Comment by ohjames** [comment]
Fixed in df9eadb and released version 0.12.11 with the fix.

**Comment by PatrikLythell** [comment]
Thanks @ohjames, that was quick!
